src/main/java/sora/util/Message.java

```diff
@@ -0,0 +1,36 @@
+package sora.util;
+
+/**
+ * Stores the messages which UI uses to print.
+ *
+ * @author Zhang Shi Chen
+ */
+public enum Message {
+    HORIZONTAL_LINE("\t____________________________________________________________\n"),
+    WELCOME("     ____________________________  ______\n"
+        + "    /  _______/  _____   / ____  \\/ _   |\n"
+        + "   /  /______   /    /  / /____/ / /_|  |\n"
+        + "  /______   /  /    /  /  _   __/  __   |\n"
+        + " _______/  /  /____/  /  / \\  \\/  /  |  |\n"
+        + "/_________/__________/__/   \\__\\_/   |__|\n"
```

Comment:
kajdnafldfjnsdf I love that you made your custom logo!!! Duke who amirite

Reply:
yeah haha, altho it might not be readable since `/` is just 1 slash but to display `\` it needs 2